README.rst

```diff
@@ -100,6 +88,67 @@ project easily.
     from mylib import logging
     logging.error('Oops, broke something.')
 
+
+Standards
+---------
+
+To help everyone standardize on the same log levels, this library
+provides a function to add some missing severity levels defined by the
+the syslog protocol in `RFC-5424 <https://tools.ietf.org/html/rfc5424>`_.
+
+This library also introduces some additional debugging levels and a
+``SUPPRESSED`` level which is intended to be used for logging suppressed
+exceptions that you may want to log, but otherwise consider handled.
+
+Use the function ``add_standards`` to add the standard levels provided
+by logging-levels:
+
+.. code:: python
+
+    from logging_levels.standards import add_standards
+    import logging
+    add_standards(logging)
+
+    log.emergency('This aggression will not stand, man.')
+    log.alert('Oh no! Something happened!')
+    log.notice('FYI this other thing happened.')
+    log.verbose('Debug, but so much more')
+    log.trace('Log every -- single -- detail')
+    log.suppressed('Warn a suppressed exception')
+
+
+All levels after using ``add_standards`` will be:
+
+EMERGENCY
++--------------+---------------+
+| Level        | Numeric Value |
++==============+===============+
+| EMERGENCY    | 100           |
++--------------+---------------+
+| ALERT        | 70            |
++--------------+---------------+
+| CRITICAL     | 50            |
++--------------+---------------+
+| ERROR        | 40            |
```

Comment:
It's confusing to include Python levels. Maybe only show the `add_standards` levels or somehow make it more clear that some of these are Python levels.